openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/EntitySummaryPanel.style.less

```diff
@@ -37,6 +37,13 @@
   -ms-overflow-style: none;
   scrollbar-width: none;
 
+  .statistics-count {
```

Comment:
Add some prefix to the className to avoid conflict

Reply:
Sure